client/components/tinymce/plugins/contact-form/dialog/field.jsx

```diff
@@ -72,15 +73,17 @@ export default React.createClass( {
 		const fielLabelValidationError = ! this.props.label;
 		const remove = <FieldRemoveButton onRemove={ this.props.onRemove } />;
 
+
 		return (
 			<FoldableCard
 				header={ <FieldHeader { ...omit( this.props, [ 'onUpdate' ] ) } /> }
 				summary={ remove }
 				expandedSummary={ remove }
-				icon="pencil"
 				expanded={ this.props.isExpanded }
 				onOpen={ () => this.props.onUpdate( { isExpanded: true } ) }
-				onClose={ () => this.props.onUpdate( { isExpanded: false } ) }>
+				onClose={ () => this.props.onUpdate( { isExpanded: false } ) }
```

Comment:
Let's try to avoid the function creation during each render, we can do something like:
```
handleCardOpen() {
    this.props.onUpdate( { isExpanded: true } );
},
handleCardClose() {
    this.props.onUpdate( { isExpanded: false } );
}
//...
render() {
    //...
    <FoldableCard 
            onClose={ this.handleCardClose }
             onOpen={ this.handleCardOpen }
}
```